battery_management_systems/lg_resu_prime_http/manifest.yml

```diff
@@ -0,0 +1,107 @@
+blueprint_spec: device/1.0
+display_name: LG RESU Prime
+icon: enapter-battery-storage
+
+communication_modules:
+  eth:
+    product: ENP-VIRTUAL
+    lua_file: firmware.lua
+
+properties:
+  model:
+    display_name: Device Model
+    type: string
+    enum:
+      - LG RESU10 Prime
+      - LG RESU16 Prime
+  ip_address:
+    display_name: IP Address
+    type: string
+
+telemetry:
+  battery_soc:
+    display_name: State of Charge
+    description: Current State of Charge
+    type: float
+    unit: percent
+  battery_energy:
+    display_name: Battery Energy
+    description: Amount Energy of the Battery
+    type: float
+    unit: watt
+  battery_voltage:
+    display_name: Battery Voltage
+    type: float
+    unit: volt
+  battery_power:
+    display_name: Power Flow
+    description: Amount of power flowing in or out of the battery
+    type: float
+    unit: watt
+  battery_temp:
+    display_name: Battery Temperature
+    description: Current temperature of the battery
+    type: float
+    unit: celsius
+  status:
+    display_name: Battery status
+    type: string
+    enum:
+      - 'Off'
+      - Standby
+      - Initializing
+      - Charging
+      - Discharging
+      - Fault
+      - Idle
+  lastresponse:
+    display_name: Last Response
+    description: Last Response from battery
+    type: string
+
+alerts:
+  lowbattery:
+    severity: error
+    code: E001
+    display_name: Low battery
+    description: Battery has less then 10% load
+
+command_groups:
+  connection:
+    display_name: Connection
+
+commands:
+  # Connection Config
+  write_configuration:
+    populate_values_command: read_configuration
+    display_name: Configure Connection
+    group: connection
+    ui:
+      icon: write_configuration
```

Comment:
There is no icon with this name in [our set](https://static.enapter.com/rn/icons/material-community.html). Please select an icon from the set, for example:
`icon: file-document-edit-outline`

Reply:
you suggested this in the first place :(

Reply:
Sorry, perhaps I didn't explain it clear at the first time. I meant that you can use **two different** icons from Enapter [icon set](https://static.enapter.com/rn/icons/material-community.html) for these commands.

Reply:
Is this actually necessary? The user should not read/write settings manually. I was hoping that the user just has to enter the IP address and the write happens automatically when they are done entering the IP address. The read-setting should happen automatically when the blueprint is running. So no manual user interaction for read/write-settings required. The code for read/write comes from one of your other blueprints but I am not sure if its required. If its NOT required (for the user to see), can I remove the yml entries?

Reply:
Also your tesla_powerwall yml uses "wrench_outline" for both read and write (which I had in the first place)...this is where I copied the base for my YML from.

Reply:
Yes, you're right, it's just optional visual improvement, but in the latest release of Enapter mobile application we added Quick Actions for commands, which can be added to site screen as simply icons (by default - with command name). Therefore, identical icons can be confusing in this case.
If you think that this is minor, then we can merge your PR.

Reply:
I rather recommend removing these two functions from the UI completely. It might confuse the user that he has to do something here. (Especially the read function as that is done automatically in the lua). 
Let me do a quick test and update it in case the read can be removed

Reply:
I suppose that the read command can't be removed?

Reply:
I removed the read-configuration from the UI there is no need for a user to do that at anytime